includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -1431,4 +1435,27 @@ private function deserialize_intention_object_from_array( array $intention_array
 
 		return $intent;
 	}
+
+	/**
+	 * Returns a formatted intention description.
+	 *
+	 * @param  int $order_id The order ID.
+	 * @return string        A formatted intention description.
+	 */
+	private function get_intent_description( int $order_id ): string {
+		if ( empty( $order_id ) ) {
```

Comment:
We can express this more specific: `if ( 0 === $order_id )`. `empty` is not so great from maintaining perspective.